### PR TITLE
Add a link to the embed view of the metric in the Slack notifications

### PR DESCRIPTION
**Pull request by madsnedergaard**
To make it easier to follow-up and see the live graph with recent changes, I think it would be nice to link to it from the Slack notifications.

Since a metric can belong to multiple dashboards, I think the easiest solution is to just use the `/embed` link as a start :)

### CODE REVIEW

File: mainapp/tasks/__init__.py [hunk 1]
```diff
@@ -234,11 +234,12 @@ def check_notify_metric_changed_task(metric_id: UUID) -> None:
         ):
             # Send slack message
             logger.info(f"Sending slack notification for metric_id={metric_id}")
+            link = f"https://polynomial.so/metrics/{metric_id}/embed"
```
Comment (corradio) [inline mainapp/tasks/__init__.py:237]: Hi @madsnedergaard,
Thanks for the contribution!
I added a new `metric-details` route [here](https://github.com/corradio/polynomial/commit/ba1997306ad6f18bbc8fab20c18e2d34cf6bac40#diff-742b0a5d9e820d2640675ab798fc9eec0ffdb9079cf908e84ee519c105614d9eR625), which is the same as `/embed` but with proper access rights and with header/footer for the UI.

I also suggest constructing the url from the named routes:
```suggestion
            link = f"{BASE_URL}{reverse('metric-details', args=[metric_id])}"
```

using:
```python
from config.settings import CSRF_TRUSTED_ORIGINS

BASE_URL = CSRF_TRUSTED_ORIGINS[0]
```

The `BASE_URL` is a bit hacky but at least it's consistent with [this](https://github.com/corradio/polynomial/blob/master/mainapp/tasks/error_handling.py#L45).

**Review by corradio — CHANGES_REQUESTED** [review]

File: mainapp/tasks/__init__.py (hunk 1 above)
Comment (corradio) [reply mainapp/tasks/__init__.py:237]: Hi @madsnedergaard , let me know if you need anything from me!

Comment (madsnedergaard) [reply mainapp/tasks/__init__.py]: Sorry it took a while, should be fixed now - I used the `reverse()` method used elsewhere, although I'm not 100% certain how that actually works ¯\\\_(ツ)\_/¯

File: Makefile [hunk 2]
```diff
@@ -1,7 +1,7 @@
 MANAGE = poetry run python manage.py
 
 seed:
-	DJANGO_SUPERUSER_USERNAME=admin \
+	DJANGO_SUPERUSER_USERNAME=john \
```
Comment (corradio) [inline Makefile:4]: john is a better name for an admin than `admin` 👍

File: mainapp/tasks/__init__.py (hunk 1 above)
Comment (corradio) [reply mainapp/tasks/__init__.py]: FYI it gets the URL by key, as defined in urls.py here:
https://github.com/corradio/polynomial/blob/master/mainapp/urls.py#L34

**Review by corradio — APPROVED** [review]

**Pull request merged**